utilities/transactions/write_unprepared_transaction_test.cc

```diff
@@ -235,20 +240,27 @@ TEST_P(WriteUnpreparedTransactionTest, RecoveryRollbackUnprepared) {
     delete iter;
   }
 
-  // Try unprepared batches with empty database.
-  for (int num_batches = 0; num_batches < 10; num_batches++) {
+  // Try unprepared batches with non-empty database.
+  for (int num_batches = 1; num_batches < 10; num_batches++) {
     // Reset database.
     prepared_trans.clear();
     ReOpen();
     wup_db = dynamic_cast<WriteUnpreparedTxnDB*>(db);
-    ASSERT_OK(db->Put(WriteOptions(), "k0", "before value"));
+    for (int i = 1; i < num_batches; i++) {
+      ASSERT_OK(db->Put(WriteOptions(), "k" + ToString(i),
+                        "before value " + ToString(i)));
+    }
 
     // Write num_batches unprepared batches into the WAL.
-    for (int i = 0; i < num_batches; i++) {
+    for (int i = 1; i < num_batches; i++) {
```

Comment:
nit: why is i changed from 0 to 1? then we writes (num_batches-1) batches instead of num_batches batches.

Reply:
Yes, should be zero. I only meant to start num_batches at 1.